nautobot/dcim/models/power.py

```diff
@@ -201,7 +201,7 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
     @property
-    def parent(self):
+    def parent_object(self):
         return self.power_panel
```

Comment:
Is this done for a reason? I see why it’s done for anything that inherits from BaseInterface but I don’t think that applies here?

Reply:
I agree with your other comment below - let's make the new FK `parent_interface` and that way we don't need to rename the existing `parent` property throughout.